cfme/fixtures/v2v_fixtures.py

```diff
@@ -378,6 +378,18 @@ def __set_conversion_instance_for_osp_ui(appliance, source_provider,  # noqa
                                        osp_ca_cert=tls_cert_key)
 
 
+def cleanup_target(provider, migrated_vm):
+    """Helper function to cleanup instances and associated volumes from openstack"""
+    if provider.one_of(OpenStackProvider):
+        volumes = []
+        for vm in provider.mgmt.list_vms():
+            if migrated_vm.name == vm.name:
+                for vol in vm.raw._info['os-extended-volumes:volumes_attached']:
+                    volumes.append(vol['id'])
```

Comment:
I guess you need to add `break` in order to stop iteration when you found necessary vm. btw, afaik there should be some special method like get_vm() which returns concrete vm .